File: 2026-03-March-GFG-POTD/March 01 - Move All Zeroes to End/Solution.java
class Solution {
     void pushZerosToEnd(int[] arr) {
        int j = 0;  // Pointer for next non-zero position
        
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] != 0) {
                // Swap arr[i] and arr[j]
                int temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
                
                j++;
            }
        }
    }
}

File: 2026-03-March-GFG-POTD/March 01 - Move All Zeroes to End/Solution.py
class Solution:
    def pushZerosToEnd(self, arr):
        j = 0  # Pointer for next non-zero position
        
        for i in range(len(arr)):
            if arr[i] != 0:
                # Swap arr[i] and arr[j]
                arr[i], arr[j] = arr[j], arr[i]
                j += 1
